OSSMDOC-198: Update configuration reference section. (#34983)
openshift/openshift-docs: modules/ossm-cr-tracing.adoc

// Module included in the following assemblies:
//
// * service_mesh/v2x/ossm-reference-smcp.adoc

[id="ossm-cr-tracing_{context}"]
= tracing parameters

> Here is an example that illustrates the `spec.tracing` parameters for the `ServiceMeshControlPlane` and a description of the available parameters with appropriate values.

Reviewer: ```suggestion
The following example illustrates the `spec.tracing` parameters for the `ServiceMeshControlPlane` object, and a description of the available parameters with appropriate values:
```

Suggestion for slight rewording of this and other similar lead ins to examples

.Example tracing parameters
[source,yaml]
----
apiVersion: maistra.io/v2
kind: ServiceMeshControlPlane
metadata:
  name: basic
spec:
  version: v2.0
  tracing:
    sampling: 100
    type: Jaeger
----

.Istio tracing parameters
[options="header"]
[cols="l, a, a, a"]
|===
|Parameter |Description |Values |Default value

|tracing:
 sampling:

> |The sampling rate determines how often the Envoy proxy generates a trace.  You  use the sampling rate to control what percentage of requests get reported to your tracing system.

Reviewer: ```suggestion
|The sampling rate determines how often the Envoy proxy generates a trace.  You can use the sampling rate to control what percentage of requests get reported to your tracing system.
```

> |Integer values between 0.01 and 100.0 in increments of 0.01%.  For example, setting the value to `10` samples 0.1% of traces, setting the value to `100` will sample 1% of traces, setting the value to `500` samples 5% of traces, and a setting of `10000` samples 100% of traces.

Reviewer 2: ```suggestion
|Integer values between 0 and 10000 representing increments of 0.01% (0 to 100%).  For example, setting the value to `10` samples 0.1% of requests, setting the value to `100` will sample 1% of requests setting the value to `500` samples 5% of requests, and a setting of `10000` samples 100% of requests.
```

|`10000`  (100% of traces)

|tracing:
 type:

> |Currently the only tracing type that is supported is `Jaeger`.  Jaeger is enabled by default.  To disable tracing, set `type` to `None`.

Reviewer: ```suggestion
|Currently the only tracing type that is supported is `Jaeger`.  Jaeger is enabled by default.  To disable tracing, set the `type` parameter to `None`.
```
